components/menu/menu-item-selectable-styles.js

```diff
@@ -8,17 +8,10 @@ export const menuItemSelectableStyles = [ menuItemStyles,
 			align-items: center;
 			display: flex;
 			padding: 0.75rem 1rem;
-
 		}
 
-		span {
```

Comment:
This is the breaking change -- from a `span` to just using the `d2l-menu-item-text` styles defined in the base shared styles.